### PR TITLE
Release notes validator will now run on: incident fields, indicator f…

### PR DESCRIPTION
**Pull request by bakatzir**
…ields, incident types, dashboard and reputations.

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/22689

## Description
minor changes in the file_validator where also needed to make sure we run the rn checks on necessary files.

## Screenshots
in the referenced issue

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/dashboard.py [hunk 1]
```diff
@@ -12,14 +12,15 @@ def get_widgets_from_dashboard(dashboard):
...
-    def is_valid_dashboard(self):
+    def is_valid_dashboard(self, validate_rn=True):
         # type: () -> bool
         """Check whether the dashboard is valid or not.
 
         Returns:
             bool. Whether the dashboard is valid or not
         """
         is_dashboard_valid = [
+            super(DashboardValidator, self).is_valid_file(validate_rn),
```
Comment (yaakovi) [inline demisto_sdk/commands/common/hook_validations/dashboard.py]: in python3 you don't need those args
```suggestion
            super().is_valid_file(validate_rn),
```

File: demisto_sdk/commands/validate/file_validator.py [hunk 2]
```diff
@@ -366,7 +366,7 @@ def validate_modified_files(self, modified_files):  # noqa: C901
 
             elif checked_type(file_path, JSON_ALL_INCIDENT_TYPES_REGEXES):
                 incident_type_validator = IncidentTypeValidator(structure_validator)
-                if not incident_type_validator.is_valid_incident_type():
+                if not incident_type_validator.is_valid_incident_type(validate_rn=True):
```
Comment (yaakovi) [inline demisto_sdk/commands/validate/file_validator.py]: why is that one different? why not `not file_type`?

Comment (bakatzir) [reply demisto_sdk/commands/validate/file_validator.py]: in `modified_files` it should always be true
in `added_files` it is `not file_type`

File: demisto_sdk/commands/common/hook_validations/dashboard.py (hunk 1 above)
Comment (bakatzir) [reply demisto_sdk/commands/common/hook_validations/dashboard.py]: @yaakovi seems like it is needed.
```
        return all(
            [
>               super.is_valid_file(validate_rn),
                self.is_valid_name(),
                self.is_valid_type(),
                self.is_valid_group(),
                self.is_valid_content_flag(),
                self.is_valid_system_flag(),
                self.is_valid_cliname(),
                self.is_valid_version(),
                self.is_current_valid_from_version(),
                self.is_valid_required()
            ]
        )
E       AttributeError: type object 'super' has no attribute 'is_valid_file'

demisto_sdk/commands/common/hook_validations/incident_field.py:182: AttributeError
```

Comment (bakatzir) [reply demisto_sdk/commands/common/hook_validations/dashboard.py]: same error occurs in the `dashboard.py` upon change.

Comment (yaakovi) [reply demisto_sdk/commands/common/hook_validations/dashboard.py]: `super()` not `super`

File: demisto_sdk/commands/validate/file_validator.py (hunk 2 above)
Comment (yaakovi) [reply demisto_sdk/commands/validate/file_validator.py]: right, my bad. sorry.

**Comment by bakatzir** [comment]
@yaakovi correct regarding the super(). changed it in all the validators.

**Pull request merged**